File: workers/dotnet/Apps/HelloWorld/HelloWorldWorkflow.cs
using Temporalio.Workflows;

namespace Temporalio.Omes.Apps.HelloWorld;

[Workflow("HelloWorldWorkflow")]
public class HelloWorldWorkflow
{
    [WorkflowRun]
    public Task<string> RunAsync(string name) => Task.FromResult($"Hello {name}");
}

File: workers/java/io/temporal/omes/apps/helloworld/HelloWorldWorkflow.java
package io.temporal.omes.apps.helloworld;

import io.temporal.workflow.WorkflowInterface;
import io.temporal.workflow.WorkflowMethod;

@WorkflowInterface
public interface HelloWorldWorkflow {
  @WorkflowMethod(name = "HelloWorldWorkflow")
  String run(String name);
}
